src/NzbDrone.Core/Download/Clients/uTorrent/UTorrentSettings.cs

```diff
@@ -55,10 +55,10 @@ public UTorrentSettings()
         [FieldDefinition(7, Label = "PostImportCategory", Type = FieldType.Textbox, Advanced = true, HelpText = "DownloadClientSettingsPostImportCategoryHelpText")]
         public string TvImportedCategory { get; set; }
 
-        [FieldDefinition(8, Label = "DownloadClientSettingsRecentPriority", Type = FieldType.Select, SelectOptions = typeof(UTorrentPriority), HelpText = "DownloadClientSettingsRecentPriorityHelpText")]
+        [FieldDefinition(8, Label = "DownloadClientSettingsRecentPriority", Type = FieldType.Select, SelectOptions = typeof(UTorrentPriority), HelpText = "DownloadClientSettingsRecentPriorityEpisodeHelpText")]
```

Comment:
```suggestion
        [FieldDefinition(8, Label = "DownloadClientSettingsRecentPriority", Type = FieldType.Select, SelectOptions = typeof(UTorrentPriority), HelpText = "DownloadClientSettingsRecentEpisodePriorityHelpText")]
```

Reply:
I tried to keep the help text related to the field title `Recent Priority` so went with `RecentPriorityEpisode` instead of `RecentEpisodePriority` even though the latter reads better. want it switched?

Reply:
That makes sense, they're just keys afterall, appreciate the reasoning!